.gitignore

```diff
@@ -1,38 +1,46 @@
 # exclude following files
 *~
+*.bak
 *.bak-mo
 *.exe
 *.exp
+*.la
+*.lo
+*.m4
+*.o
 *.pyc
+.deps
+.dirstamp
+.libs
+Modelica/Resources/BuildProjects/VisualStudio2005/*.filters
+Modelica/Resources/BuildProjects/VisualStudio2005/*.vcxproj
+Modelica/Resources/BuildProjects/autotools/AUTHORS
+Modelica/Resources/BuildProjects/autotools/COPYING
+Modelica/Resources/BuildProjects/autotools/ChangeLog
+Modelica/Resources/BuildProjects/autotools/INSTALL
+Modelica/Resources/BuildProjects/autotools/Makefile*
+Modelica/Resources/BuildProjects/autotools/NEWS
+Modelica/Resources/BuildProjects/autotools/compile
```

Comment:
Why is this necessary. Why would one add all those files to the repo to begin with?

Reply:
It's already there. Just resorted alphabetically.

Reply:
Why not removing it then. Seems to make the .gitignore unnecessary bloated.

Reply:
I guess @sjoelund added them by purpose in #2933.

Reply:
Also, it does not hurt ignoring them.

Reply:
They are required by the automake stuff. The script generates them.

Reply:
OK, I see.